test/circuit.js

```diff
@@ -0,0 +1,290 @@
+/* eslint max-nested-callbacks: ["error", 8] */
+/* eslint-env mocha */
+'use strict'
+
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+const expect = chai.expect
+chai.use(dirtyChai)
+
+const parallel = require('async/parallel')
+const series = require('async/series')
+
+const isNode = require('detect-node')
+
+const utils = require('./utils/circuit')
+
+const getProc = utils.setUpProcNode
+const getJs = utils.setUpJsNode
+const getGo = utils.setUpGoNode
```

Comment:
You prefixed `get` the wrong ones. See my comment https://github.com/ipfs/interop/pull/6#discussion_r172467235